pinot-common/src/main/java/org/apache/pinot/common/function/FunctionRegistry.java

```diff
@@ -217,5 +217,30 @@ public static Object clpDecode(String logtypeFieldName, String dictVarsFieldName
     public static String arrayToMV(Object multiValue) {
       throw new UnsupportedOperationException("Placeholder scalar function, should not reach here");
     }
+
+    @ScalarFunction(names = "arrayMin", isPlaceholder = true)
```

Comment:
did we add any tests for this? unit or integraiton tests?

Reply:
Sorry I missed this comment. We added unit tests in a different pr here https://github.com/apache/pinot/pull/11498/files#diff-9724fd2cc0075a3c9c9a29cc7b98bb921cfe2690c2cc0b6e0ad47dec0589e6abR630

Reply:
no worries. was my retro review after the merge. resolving